### PR TITLE
✨ specify scoring system via name

### PR DESCRIPTION
**Pull request by arlimus**
image

Signed-off-by: Dominik Richter <dominik.richter@gmail.com>

### CODE REVIEW

File: policy/policy.go [hunk 1]
```diff
@@ -556,3 +559,27 @@ func IsPolicyMrn(candidate string) error {
...
+	var code int32
+	err := json.Unmarshal(data, &code)
+	if err == nil {
+		*s = ScoringSystem(code)
+	} else {
+		var name string
+		_ = json.Unmarshal(data, &name)
+
+		switch name {
+		case "lowest":
```
Comment (chris-rock) [inline policy/policy.go]: Why not "worst"?

**Review by chris-rock — APPROVED** [review]
human readable scoring system, yes

File: policy/policy.go (hunk 1 above)
Comment (arlimus) [reply policy/policy.go]: Sounded like "worst scoring system". I've always regretted that name ever since choosing it, but I may also be wrong about it...

Comment (arlimus) [reply policy/policy.go]: Changed based on more feedback, see new screenshot. Thank you for raising it!

**Review by chris-rock — APPROVED** [review]
Thank you @arlimus

**Pull request merged**